README.md

```diff
@@ -4,48 +4,61 @@ This repo contains the Docker file used in the [AiiDA lab](https://www.materials
 
 Docker images are available from Dockerhub via `docker pull aiidalab/aiidalab-docker-stack:latest`.
 
-## Deploy
+# Deploy on AiiDA lab server
+
 To deploy changes, log into the AiiDA lab server and execute the following commands:
 ```
-cd /home/ubuntu/aiidalab-docker-stack/
-git pull
-./build.sh
-./inspect.sh  (optionally)
-./activate.sh
+docker pull aiidalab/aiidalab-docker-stack:latest
+docker tag aiidalab/aiidalab-docker-stack:latest aiidalab-docker-stack:latest
 ```
-
 The users will gradually pick up the new image, whenever they restart their container via the _Control Panel_.
 
-## Cheat Sheet
+# Deploy locally
+
+To run AiiDA lab image image locally, you first need to make sure that [Docker](https://www.docker.com/) is installed on your computer. Run `$ docker` in the command line and see if you get something like
+```
+Usage:	docker [OPTIONS] COMMAND
+
+A self-sufficient runtime for containers
+
+Options:
+...
+```
+
+If this is the case, everything proceed with the next steps. Otherwise, please visit the [Docker installation page](http://www.docker.com/install) and install Docker on your machine.
+
+To pull image from DockerHub and start the container do the following:
+```
+docker pull aiidalab/aiidalab-docker-stack:latest
+mkdir ${HOME}/aiidalab # Create a new folder where your data vill be stored
+docker run -d -p 8888:8888 -v ${HOME}/aiidalab:/home/aiida aiidalab/aiidalab-docker-stack:latest
+# Note the Docker ID that was displayed after the last command and put it into the following run
```

Comment:
Wouldn't it be easier to encapsulate the steps from `docker run` in a script?

Reply:
done.